src/components/my-career/SkillsRecommendationCourses.jsx

```diff
@@ -0,0 +1,132 @@
+import React, {
+  useContext, useMemo, useState, useEffect,
+} from 'react';
+import { useHistory } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { AppContext } from '@edx/frontend-platform/react';
+import { CardGrid, Hyperlink } from '@edx/paragon';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import SearchCourseCard from '../search/SearchCourseCard';
+
+import { useDefaultSearchFilters, useSearchCatalogs } from '../search/data/hooks';
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { SubsidyRequestsContext } from '../enterprise-subsidy-requests';
+
+const SkillsRecommendationCourses = ({ index, subCategoryName, subCategorySkills }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  const {
+    subscriptionPlan, subscriptionLicense, couponCodes: { couponCodes }, enterpriseOffers,
+  } = useContext(UserSubsidyContext);
+  const { catalogsForSubsidyRequests } = useContext(SubsidyRequestsContext);
+  const history = useHistory();
+
+  const searchCatalogs = useSearchCatalogs({
+    subscriptionPlan,
+    subscriptionLicense,
+    couponCodes,
+    enterpriseOffers,
+    catalogsForSubsidyRequests,
+  });
+
+  const { filters } = useDefaultSearchFilters({
+    enterpriseConfig,
+    searchCatalogs,
+  });
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [courses, setCourses] = useState([]);
+  const [hitCount, setHitCount] = useState(undefined);
+
+  const skillsFacetFilter = useMemo(
+    () => {
+      if (subCategorySkills.length > 0) {
+        return subCategorySkills.map((skill) => `skill_names:${skill}`);
+      }
+      return [];
+    },
+    [subCategorySkills],
+  );
+  const moreCoursesLink = useMemo(
+    () => {
+      if (subCategorySkills) {
+        const enterpriseSlug = enterpriseConfig.slug;
+        const searchParameters = 'showAll=1&content_type=course';
+        const skillNames = subCategorySkills.join('&skill_names=');
+        return `/${enterpriseSlug}/search?${searchParameters}&skill_names=${skillNames}`;
```

Comment:
I think it would be better to use `URL` class to construct this URL. Ref: https://javascript.info/url